src/plugins/Wallet/erc20.ts

```diff
@@ -8,5 +8,8 @@ export interface ERC20Token {
 export type ERC20TokenPredefinedData = ERC20Token[]
 
 export const ETH_ADDRESS = '0x0000000000000000000000000000000000000000'
+export const GITCOIN_ETH_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
```

Comment:
?

Reply:
This is the virtual address used in Gitcoin for standing ETHER token when doing donation.